docs/Project.toml

```diff
@@ -1,12 +1,11 @@
 [deps]
 Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
-Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
 ColorSchemes = "35d6a980-a343-548e-a6ea-1d62b119f2f4"
+Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
 Compose = "a81c6b42-2e10-5240-aca2-a61377ecd94b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-Gadfly = "c91e804a-d5a3-530f-b6f0-dfbca275c004"
```

Comment:
This deletion was necessary to build the docs locally, or Julia would just install the one from the general registry. In general, it is not advised to include the package in `docs/Project.toml` as is, as chance would have it, also discussed at <https://discourse.julialang.org/t/error-when-building-documentation-with-github-actions/56139/2>. Note that CI installs the dev version via the default procedure
```
julia --project=docs -e '
    using Pkg
    Pkg.develop(PackageSpec(path=pwd()))
    Pkg.instantiate()'
```